PWGHF/DataModel/HFSecondaryVertex.h

```diff
@@ -803,7 +803,64 @@ auto InvMassXToJpsiPiPi(const T& candidate)
 {
   return candidate.m(array{RecoDecay::getMassPDG(443), RecoDecay::getMassPDG(kPiPlus), RecoDecay::getMassPDG(kPiPlus)});
 }
+template <typename T>
+auto QX(const T& candidate)
+{
```

Comment:
Add documentation and blank lines to separate functions.

Reply:
Please make it a proper Doxygen documentation, starting with `///`.
Fix the other functions as well.